.github/workflows/build_win.yml

```diff
@@ -19,11 +19,11 @@ jobs:
         pip install meson==0.58.0 ninja
     - name: Build
       run: |
-        meson --backend=ninja build -Dlog=true -Dloaders="all" -Dsavers="all" -Dbindings="capi" -Dtools="all"
+        meson --backend=ninja build -Dlog=true -Dloaders="tvg, svg, png, jpg" -Dsavers="all" -Dbindings="capi" -Dtools="all"
```

Comment:
We need to install libwebp separately in the window container, but I haven't prepared it yet.

Reply:
We can keep it all but not break the build. It just could try all the valid options at the time.

Reply:
The reason the build don't stop is that if there is no external loader, the static loader is used instead.
Since our webp does not provided a static loader, it is logical that the build breaks without libwebp. (In this case, it is because webp(or all) is explicitly enabled even though static is not specified and libwebp dependency exists.)

Nevertheless, if we want to keep the build success for `all` options without libwebp, I can move libwebp's dependency check to before declaring `THORVG_WEBP_LOADER_SUPPORT`.

Please hope this is the solution.

Reply:
I see, please keep your change now, we can revisit this later. it's not important at this moment.

Reply:
Please review the additional comments and then make a decision on whether you think any further changes are necessary.